meta/saisanitycheck.c

```diff
@@ -1856,8 +1861,13 @@ void check_attr_acl_fields(
         case SAI_ATTR_VALUE_TYPE_ACL_ACTION_DATA_OBJECT_ID:
         case SAI_ATTR_VALUE_TYPE_ACL_ACTION_DATA_OBJECT_LIST:
 
-            if (md->objecttype != SAI_OBJECT_TYPE_ACL_ENTRY ||
-                    md->attrid < SAI_ACL_ENTRY_ATTR_ACTION_START ||
+			if (md->objecttype == SAI_OBJECT_TYPE_ACL_ENTRY && md->isextensionattr)
```

Comment:
can you fix ident ?

Reply:
Done! @kcudnik could you please check now ?